psi4/src/read_options.cc

```diff
@@ -1937,11 +1939,11 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     options.add_int("CACHELEVEL",2);
     /*- Specifies the choice of representation of the electric dipole operator.
     Acceptable values are ``LENGTH`` for the usual length-gauge representation,
-    ``VELOCITY`` for the modified velocity-gauge representation in which the
+    ``VELOCITY``(default) for the modified velocity-gauge representation in which the
     static-limit optical rotation tensor is subtracted from the frequency-
     dependent tensor, or ``BOTH``. Note that, for optical rotation calculations,
     only the choices of ``VELOCITY`` or ``BOTH`` will yield origin-independent results. -*/
-    options.add_str("GAUGE","LENGTH", "LENGTH VELOCITY BOTH");
+    options.add_str("GAUGE","VELOCITY", "LENGTH VELOCITY BOTH");
```

Comment:
Thanks for the feedback. Our ccresponse module calculates three properties right now: optical-rotation, dipole-polarizability and raman optical activity. The intention was to change the gauge for rotation and raman optical activity. For other properties like polarizabilities we only calculate the length gauge version in psi4 irrespective of the gauge provided in input file.